.github/workflows/tests.yml

```diff
@@ -57,3 +57,20 @@ jobs:
     - name: Run tests
       run: pytest tests/
 
+  tests38:
```

Comment:
This could be done with a matrix for the python version instead of duplicating the code. But as we want to get rid of Python 2 in the near future, I guess this won't have a long life anyway... 😄

Reply:
oh, good point. though, then the linting would happen twice?

Reply:
@williballenthin 
> though, then the linting would happen twice?

no, I meant changing `tests` to the following (instead of duplicating  `tests` into `tests27` and `test38`:

```
  tests:
	name: Test in ${{ matrix.python }}
    runs-on: ubuntu-latest
    needs: [code_style, rule_linter]
    strategy:
      matrix:
        include:
          - python: 2.7
		  - python: 3.8
    steps:
    - name: Checkout capa with submodules
      uses: actions/checkout@v2
      with:
        submodules: true
    - name: Set up Python ${{ matrix.python }}
      uses: actions/setup-python@v2
      with:
        python-version: ${{ matrix.python }}
    - name: Install capa
      run: pip install -e .[dev]
    - name: Run tests
      run: pytest tests/
```
I can send a PR as this can be useful if we want to test several python versions.